src/components/Breadcrumb/Breadcrumb.js

```diff
@@ -0,0 +1,33 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+class Breadcrumb extends Component {
+	render() {
+		const { history } = this.props;
+
+		return (
+			<div className="breadcrumb">
+				{history.map((item, index) => {
+					return (
+						<React.Fragment key={index}>
+							{item.isActive ? (
+								<div className="breadcrumb--object">{item.title}</div>
```

Comment:
This is the element inside the block of the 'breadcrumb' right? so it should be breadcrum__object

we '--' used this for the modifiers like the sizes, colors, actions.